blazor/chart/chart-print.md

```diff
@@ -288,6 +289,109 @@ The optional parameters for this method are,
     };
 }
 
+```
+
+### Export Chart as base64 string
+
+The image can be exported as base64 string for the JPEG, PNG and PDF formats. The rendered Chart can be exported to image as base64 string using the [OnExportComplete](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.ChartEvents.html#Syncfusion_Blazor_Charts_ChartEvents_OnExportComplete) event. The arguments that are required for this method is image type, file name, orientation of the exported PDF document which must be set as null for image export and 0 or 1 for the PDF export and finally allowDownload which should be set as false to return base64 string. 
+
+The following code shows how to pass parameters to the `ExportAsync` method to get `base64` string.
+
+In this method, you'll need to specify the following parameters:
+
+* [Export Type](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.ExportType.html): The desired export format (e.g., Png, Jpeg, Svg, Pdf).
+* `fileName`: A name for the exported file (this is not used when exporting as base64).
+* `allowDownload`: Set this parameter to false to prevent the browser's download prompt.
+* `isBase64`: Set this parameter to true to indicate that you want to receive the exported content as a base64 string.
```

Comment:
"The following code demonstrates how to pass parameters to the ExportAsync method in order to obtain a base64 string.

In this method, you'll need to specify the following parameters:

Export Type: The desired export format (e.g., Png, Jpeg, Svg, Pdf).
fileName: A name for the exported file (this is not used when exporting as a base64 string).
allowDownload: Set this parameter to false to prevent the browser's download prompt.
isBase64: Set this parameter to true to indicate that you want to receive the exported content as a base64 string."

Use this